grpc-gcp/src/main/java/com/google/cloud/grpc/GcpManagedChannel.java

```diff
@@ -363,6 +493,20 @@ private void initMetrics() {
         this,
         GcpManagedChannel::reportMaxReadyChannels);
 
+    createDerivedLongGaugeTimeSeries(
+        GcpMetricsConstants.METRIC_NUM_CHANNELS,
+        "The number of channels currently in the pool.",
+        GcpMetricsConstants.COUNT,
+        this,
+        GcpManagedChannel::reportNumChannels);
+
+    createDerivedLongGaugeTimeSeries(
+        GcpMetricsConstants.METRIC_MIN_CHANNELS,
+        "The minimum number of channels in the pool.",
```

Comment:
Isn't this a constant, i.e., `minSize`?

Reply:
Yeah question is that why you need a metrics to monitor a constant given a process?

Reply:
It is a constant only within a lifetime of a single process. Imagine you run it with one value and then changed configuration, restarted processes. With this metric you don't need to remember when you changed the configuration to what. If you look at the number of channels metrics and wonder "why it didn't scale down?", then you can look at this metric and verify "oh, it was already at the minimum channels". Or imagine you are running different workloads with different configuration and ship metrics to a central place, where you can aggregate it differently, then it will be also useful to have pool size boundaries there.

Reply:
I understand, but it is still weird. A constant to a process is basically a flag, and I feel it is weird to set up a metrics for a flag.